qctrlopencontrols/dynamic_decoupling_sequences/driven_controls.py

```diff
@@ -95,7 +95,7 @@ def _check_maximum_rotation_rate(
                     'allowed_maximum_detuning_rate': UPPER_BOUND_DETUNING_RATE})
 
 
-def convert_dds_to_driven_controls(
+def convert_dds_to_driven_control(
```

Comment:
Same story here -- if an external package is using the old method, they'll now be broken (which might be fine, I'm just not sure).

Reply:
This is a valid concern. This method was public. IMHO - we should be more consistent in naming from the start.